schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+id INT,
```

Comment:
[OPTIONAL]
- You can make the id column not null and autoincremented for example:
`id int NOT NULL IDENTITY(1,1)`

- You can also mark the id column as primary key for example:
`PRIMARY KEY (id)`